app/controllers/superpowers_controller.rb

```diff
@@ -0,0 +1,55 @@
+class SuperpowersController < ApplicationController
+  before_action :authenticate_user!, only: [:new, :create]
```

Comment:
we do want the user to be authenticated on :edit, :update, and :destroy, too, right?